src/common/lib/command_mode.c

```
@@ -38,129 +39,128 @@
+static uint8_t  brightness_original_value = 0; /**< Original brightness for selection mode */
+static uint16_t brightness_rainbow_hue = 0; /**< Current rainbow hue (0-359) for visual feedback */
 #endif
 
 /** Command mode timing constants (milliseconds) */
-#define CMD_MODE_HOLD_TIME_MS 3000      /**< Time to hold command keys to enter command mode */
-#define CMD_MODE_TIMEOUT_MS 3000        /**< Timeout for all command mode states before returning to idle */
-#define CMD_MODE_LED_TOGGLE_MS 100      /**< LED toggle interval in command mode */
+#define CMD_MODE_HOLD_TIME_MS   3000 /**< Time to hold command keys to enter command mode */
+#define CMD_MODE_TIMEOUT_MS     3000 /**< Timeout for command mode states before returning to idle */
+#define CMD_MODE_LED_TOGGLE_MS  100  /**< LED toggle interval in command mode */
+#define CMD_MODE_BRIGHTNESS_MAX 10   /**< Maximum LED brightness step */
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)